presto-docs/src/main/sphinx/functions/qdigest.rst

```diff
@@ -30,6 +30,11 @@ Functions
     Returns the approximate percentile values from the quantile digest given
     the number ``quantile`` between 0 and 1.
 
+.. function:: quantile_at_value(qdigest(T), T) -> quantile
+
+    Returns the approximate ``quantile`` number between 0 and 1
+    from the quantile digest given an input value.
```

Comment:
Returns the approximate ``quantile`` number between 0 and 1 from the quantile digest given an input value. If quantile digest is empty, return 0.